app/models/file_depot.rb

```diff
@@ -16,6 +16,11 @@ def self.supported_protocols
     @supported_protocols ||= subclasses.each_with_object({}) { |klass, hash| hash[klass.uri_prefix] = klass.name }.freeze
   end
 
+  def self.depot_description_to_class(description)
+    class_name = supported_depots.key(description)
```

Comment:
Glad you found a way to make brakeman happy with this. I feel like I have to jump through great lengths to have brakeman feel like I'm not doing something wrong here.

Hmm. wonder if brakeman understands `try` and would complain about my other suggestion.